### ISSUE TITLE
Breaking change rule output when an operation is removed

### ISSUE THREAD

**Issue by cmars** [snyk/sweater-comb#79]
This breaking change scenario fails, but the output is not very clear about what needs fixing.

Steps to reproduce:

```
$ node_modules/.bin/ts-node src/index.ts compare \
  --from end-end-tests/api-standards/resources/thing/2021-11-10/000-baseline.yaml \
  --to end-end-tests/api-standards/resources/thing/2021-11-10/001-fail-operation-removed.yaml \
  --context '{"changeDate":"2021-11-11","changeResource":"thing","changeVersion":"2021-11-10"}' 2>&1 | pbcopy
```

Output:

```
Loading specifications for comparison:
Current specification: loading...
Next specification: loading...
[2K[1A[2K[1A[2K[1A[2K[GLoading specifications for comparison:
Current specification: done
Next specification: loading...
[2K[1A[2K[1A[2K[1A[2K[GLoading specifications for comparison:
Current specification: done
Next specification: done
running rules...
[2K[1A[2K[1A[2K[1A[2K[1A[2K[GLoading specifications for comparison:
Current specification: done
Next specification: done


  PASS  DELETE /thing/{thing_id}


  PASS  GET /thing


  PASS  GET /thing/{thing_id}


  FAIL  PATCH /thing/{thing_id}
  x removed response: undefined must not be removed
    Cannot read property 'statusCode' of undefined
  x removed response: undefined must not be removed
    Cannot read property 'statusCode' of undefined
  x removed response: undefined must not be removed
    Cannot read property 'statusCode' of undefined
  x removed response: undefined must not be removed
    Cannot read property 'statusCode' of undefined
  x removed response: undefined must not be removed
    Cannot read property 'statusCode' of undefined
  x removed response: undefined must not be removed
    Cannot read property 'statusCode' of undefined
  x removed response: undefined must not be removed
    Cannot read property 'statusCode' of undefined
  x removed response: undefined must not be removed
    Cannot read property 'statusCode' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined
  x removed field: undefined must not be removed
    Cannot read property 'key' of undefined


  PASS  POST /thing





1051 checks passed
88 checks failed


```

**Comment by cmars** [comment]
CC @acunniffe

**Comment by cmars** [comment]
With #80 landed, now the expected rule failure is passing. An operation being removed should still be a breaking change:

```
[2J[3J[H[1m[34mLoading specifications for comparison:[39m[22m
[37mCurrent specification: [1m[32mloading...[39m[37m[22m[39m
[37mNext specification: [1m[32mloading...[39m[37m[22m[39m[?25l[2J[3J[H[1m[34mLoading specifications for comparison:[39m[22m
[37mCurrent specification: [1m[32mloading...[39m[37m[22m[39m
[37mNext specification: [1m[32mdone[39m[37m[22m[39m[2J[3J[H[1m[34mLoading specifications for comparison:[39m[22m
[37mCurrent specification: [1m[32mdone[39m[37m[22m[39m
[37mNext specification: [1m[32mdone[39m[37m[22m[39m[2J[3J[H[1m[34mLoading specifications for comparison:[39m[22m
[37mCurrent specification: [1m[32mdone[39m[37m[22m[39m
[37mNext specification: [1m[32mdone[39m[37m[22m[39m
running rules...[2J[3J[H[1m[34mLoading specifications for comparison:[39m[22m
[37mCurrent specification: [1m[32mdone[39m[37m[22m[39m
[37mNext specification: [1m[32mdone[39m[37m[22m[39m


 [1m[42m PASS [49m[22m [37m[1mDELETE[22m /thing/{thing_id}[39m


 [1m[42m PASS [49m[22m [37m[1mGET[22m /thing[39m


 [1m[42m PASS [49m[22m [37m[1mGET[22m /thing/{thing_id}[39m


 [1m[42m PASS [49m[22m [37m[1mPOST[22m /thing[39m





[1m[32m1051 checks passed[39m[22m
[1m[31m0 checks failed[39m[22m[2J[3J[H[1m[34mLoading specifications for comparison:[39m[22m
[37mCurrent specification: [1m[32mdone[39m[37m[22m[39m
[37mNext specification: [1m[32mdone[39m[37m[22m[39m


 [1m[42m PASS [49m[22m [37m[1mDELETE[22m /thing/{thing_id}[39m


 [1m[42m PASS [49m[22m [37m[1mGET[22m /thing[39m


 [1m[42m PASS [49m[22m [37m[1mGET[22m /thing/{thing_id}[39m


 [1m[42m PASS [49m[22m [37m[1mPOST[22m /thing[39m





[1m[32m1051 checks passed[39m[22m
[1m[31m0 checks failed[39m[22m[?25h[?25h[?25h
```

**Comment by cmars** [comment]
This was fixed in #87 (thank you!)
